src/scripts/main.js

```diff
@@ -1 +1,14 @@
 'use strict';
+
+const largeImg = document.querySelector('#largeImg');
+const listItem = document.querySelectorAll('#thumbs a');
```

Comment:
Here, you are using `querySelectorAll` which returns a NodeList. If there are no elements found, it will return an empty NodeList. However, if you try to call `forEach` on an undefined or null, it will throw an error. So it's a good practice to check if `listItem` is not empty before calling `forEach` on it.